src/fetchers/fetchEmployees.ts

```diff
@@ -0,0 +1,45 @@
+import pMap from 'p-map';
+import sdk from '@api/bamboohr'; // @bamboohr/v1#43haj4kl796vw3d
+
+import { CONCURRENCY } from '../common';
+import {
+  TBambooHREmployee,
+  TBambooHREmployeeExtended,
+  TBambooHREmployeesDirectory,
+} from '..';
+import commonMetadata from './commonMetadata';
+
+async function mapper(
+  employee: TBambooHREmployee
+): Promise<TBambooHREmployeeExtended> {
+  const { data } = await sdk.getEmployee({
+    ...commonMetadata,
+    id: employee.id,
+    fields: 'birthday,hireDate,country',
+  });
+
+  return {
+    ...employee,
+    ...data,
+  };
+}
+
+export default async function fetchEmployees(
+  countryList: string[]
+): Promise<TBambooHREmployeeExtended[]> {
+  let employees: TBambooHREmployeeExtended[];
+
+  const { data } = await sdk.getEmployeesDirectory({
+    ...commonMetadata,
+  });
+
+  employees = await pMap(
+    (data as TBambooHREmployeesDirectory).employees,
+    mapper,
+    { concurrency: CONCURRENCY }
+  );
+
+  return countryList.length > 0
+    ? employees.filter(employee => countryList.includes(employee.country ?? ''))
```

Comment:
Just a question. Why use the `country`?

Reply:
This is to allow sending birthday notifications only for the configured countries because some countries don't want their birthdays to be notified.

Reply:
Wouldn't be more suitable to use the Bamboo's `division` in that case?

Reply:
I don't think so. My division in BambooHR is `Development Spain`, so we would need to add a lot of them (Human Resources, Delivery, ...)

Reply:
Yep, I know.

The `country` field actually indicates the _division_ within the company, or the home country of the user? If the second, you would also have to manage several potential countries.

I use the `division` field with a RegExp in the graph tool. If you are interested and fits the app use case, I'd be happy to show you. Otherwise, feel free to go ahead with this solution 👍